Editor/Scripts/UserSettings/DCCToolsSettingsTab.cs

```diff
@@ -124,20 +124,48 @@ internal class DCCToolsSettingsTab : IMeshSyncSettingsTab{
 //----------------------------------------------------------------------------------------------------------------------        
 
     #region Button callbacks
+
+    private const string addDCCToolLabel = "Add DCC Tool";
+    
+    string GetFilePath() {
+        var path = EditorUtility.OpenFilePanel(addDCCToolLabel, m_lastOpenedFolder, "");
+        
+        if (!string.IsNullOrEmpty(path)) {
+            m_lastOpenedFolder = Path.GetDirectoryName(path);
+        }
+        
+        return path;
+    }
+
+    string GetFolderPath() {
```

Comment:
Can you please change the name to indicate that this opens a FolderPanel ?

Reply:
done